packages/react-components/react-portal-compat-context/etc/react-portal-compat-context.api.md

```diff
@@ -3,10 +3,11 @@
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
+
 import * as React_2 from 'react';
 
 // @public (undocumented)
-export const PortalCompatContextProvider: React_2.Provider<RegisterPortalFn>;
+export const PortalCompatContextProvider: React_2.Provider<RegisterPortalFn | undefined>;
```

Comment:
This does not seem correct to me, can you please type cast it to a previous value? (otherwise it's possible to do `<PortalCompatContextProvider value={undefined} />`

Reply:
You're right, the type casting is a good idea 👍 I've done that for all changes in this PR